source/Phage.test.cc

```diff
@@ -283,92 +313,60 @@ TEST_CASE("Phage process"){
 
             p->Process(location);
 
-            THEN("The phage is set to dead"){
+            THEN("The phage doesn't induce and is set to dead"){
                 REQUIRE(p->GetDead() == expected_dead);
             }
 
             h.Delete();
         }
-        
-    }
 
-    WHEN("The phage chooses lysis"){

```

Comment:
Was this test moved, if so where? It looks like this new test is testing something different, so I want to make sure we didn't lose the "When the phage chooses lysis" test

Reply:
Looks like it was moved to HostPhageUnitTest and the name changed